src/cf-icons/src/cf-icons.less

```diff
@@ -38,6 +38,20 @@
         // text stand on its own
         display: none;
     }
+
+    &__updating {
+        animation: updating-animation 1.25s infinite linear;
+        transform-origin: 50% 50%;
+    }
+}
+
+@keyframes updating-animation {
+    0% {
+        transform: rotate( 0deg );
+    }
+    100% {
+        transform: rotate( 359deg );
```

Comment:
I feel like this should be `transform: rotate( 360deg );`, though I haven't seen a case made either way. Conceptually, only going to 359 would mean that there'd be a 1 degree jump at the start of the loop (from 359 to 0), but I know there are certain animation frame hacks that require dropping a degree to make the end result seem smoother, but I dunno if that is true in this case.

Reply:
No, 360 and 0 are the same position, so if you do 360 there's a pause at the end rather than smooth animation.